.travis.yml

```diff
@@ -7,17 +7,25 @@ script:
 matrix:
   include:
   - osx_image: xcode11.2
-    env: ACTION="swift-package";SDK="iphonesimulator";DESTINATION="platform=iOS Simulator,OS=13.2.2,name=iPad Pro (12.9-inch) (3rd generation)";SHOULD_TEST="true"
+    env: ACTION="swift-package";PLATFORMS="iOS_12,iOS_13";
+
   - osx_image: xcode11.2
-    env: ACTION="swift-package";SDK="appletvsimulator13.2";DESTINATION="platform=tvOS Simulator,name=Apple TV";SHOULD_TEST="true";ENABLE_CODE_COVERAGE="true"
```

Comment:
While I was here I simplified the configuration in this file, and pushed the complexity to the easier to read `build.swift` file. (I originally thought I had to change this system to get code coverage reports merged, but it turns out I didn't 🤦‍♂️)